site/gatsby-site/src/components/forms/SubmissionWizard/StepOne.js

```diff
@@ -321,7 +321,11 @@ const FormDetails = ({
               value={values.text}
               onChange={(value) => {
                 setFieldValue('text', value);
-                setFieldTouched('text', true);
+                if (!submissionComplete) {
```

Comment:
What's this for, looks very specific, like we are misusing formik 🤔

Reply:
Unfortunately, formik does not work well with the `Editor component` and the function `resetForm` doesn't reset tits errors, and touched values.
So this entails that once the form is submitted the text editor appears surrounded by red with the text "Editor is required" because it has not been reset by Formik.